src/main/python/deep_qa/training/text_trainer.py

```diff
@@ -579,4 +579,11 @@ def _get_custom_objects(cls):
         for value in encoders.values():
             if value.__name__ not in ['LSTM']:
                 custom_objects[value.__name__] = value
+        custom_objects["TimeDistributedEmbedding"] = TimeDistributedEmbedding
+
+        # These are used in the words_and_characters tokenizer.
```

Comment:
You could leave a note here that we might consider making the Tokenizer API return custom objects.